plopfile.js

```diff
@@ -0,0 +1,175 @@
+        actions.push({
+          type: 'add',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/pages/api/hello.ts`,
+          templateFile: `plop-templates/example/pages/api/hello.ts`,
+        })
+      }
+
+      if (data.options.includes('middleware')) {
+        actions.push({
+          type: 'add',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/pages/_middleware.ts`,
+          templateFile: `plop-templates/example/pages/_middleware.ts`,
+        })
+      }
+
+      if (data.options.includes('tailwind')) {
+        // Tailwind files
+        TailwindFiles.forEach((file) => {
+          actions.push({
+            type: 'add',
+            path: `{{exampleScopeFolder}}/${plopExampleName}/${file}`,
+            templateFile: `plop-templates/example/${file}`,
+          })
+        })
+      } else {
+        // remove tailind deps
+        actions.push({
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/package.json`,
+          transform: (fileContents, data) => {
+            const packageData = JSON.parse(fileContents)
+            const removePackages = ['autoprefixer', 'postcss', 'tailwindcss']
+            packageData.devDependencies = Object.keys(
+              packageData.devDependencies
+            )
+              .filter((package) => !removePackages.includes(package))
+              .reduce((obj, key) => {
+                obj[key] = packageData.devDependencies[key]
+                return obj
+              }, {})
+
+            return JSON.stringify(packageData, null, 2)
+          },
+        })
+      }
+
+      return [
+        ...actions,
+        // README.md
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/README.md`,
+          pattern: /(-- PLOP TITLE HERE --)/gi,
+          template: `${data.name}`,
+        },
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/README.md`,
+          pattern: /(-- PLOP EXAMPLE NAME HERE --)/gi,
+          template: `${plopExampleName}`,
+        },
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/README.md`,
+          pattern: /(-- PLOP PATH HERE --)/gi,
+          template: `${plopPath}`,
+        },
+        // package.json
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/package.json`,
+          pattern: /(-- PLOP EXAMPLE NAME HERE --)/gi,
+          template: `${plopExampleName}`,
+        },
+        // _app.tsx
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/pages/_app.tsx`,
+          pattern: /(-- PLOP TITLE HERE --)/gi,
+          template: `${data.name}`,
+        },
+        {
+          type: 'modify',
+          path: `{{exampleScopeFolder}}/${plopExampleName}/pages/_app.tsx`,
+          pattern: /(-- PLOP PATH HERE --)/gi,
```

Comment:
Awesome! Probably we can replace this text with something else? Like [[:ploppath:]] ?